web/components/issues/issue-layouts/filters/applied-filters/roots/project-view-root.tsx

```diff
@@ -91,6 +91,12 @@ export const ProjectViewAppliedFiltersRoot: React.FC = observer(() => {
       filters: {
         ...(appliedFilters ?? {}),
       },
+      display_filters: {
```

Comment:
why are we spreading it here?

Reply:
Since they are coming directly from the store, I made it a habit to change reference when trying to process the data. I have seen few instances where the store object gets mutated unknowingly.